test/integration/targets/canonical-pep517-self-packaging/runme.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
```

Comment:
This script might be shorter and easier to work with if it was written in Python.

Reply:
Yeah, I did consider this. I've already had some boilerplate, so I've gone with it. I think it's okay for the first iteration and happy to rewrite it based on the feedback.